master/getting-started/kubernetes/upgrade/index.md

```diff
@@ -4,23 +4,36 @@ redirect_from: latest/getting-started/kubernetes/upgrade
 no_canonical: true
 ---
 
+Upgrading to this release of Calico from a pre-v3.0 release may require some manual steps, depending on
+your choice of datastore.
 
-## Prerequisites
+## Before you begin
 
 - You must [upgrade](/v2.6/getting-started/kubernetes/upgrade) 
-  to a tagged release of {{site.prodname}} [v2.6.4](https://github.com/projectcalico/calico/releases) before you can upgrade to {{site.prodname}} {{site.data.versions[page.version].first.title}}. 
-
-- An [etcdv3 cluster](https://coreos.com/etcd/docs/latest/). 
-
+  to a release of {{site.prodname}} [v2.6.4](https://github.com/projectcalico/calico/releases) 
+  (or a later v2.6.x release) before you can upgrade to {{site.prodname}} 
+  {{site.data.versions[page.version].first.title}}. 
+  
+  > **Important**: Calico v2.6.4 was a special transitional release that included changes to enable 
+  > upgrade to v3.0.0+; do not skip this step!
+  {: .alert .alert-danger}
+
+- If you are using the etcd datastore, you should upgrade etcd to the latest stable 
+  [etcdv3 release](https://coreos.com/etcd/docs/latest/). 
+  
+  > **Important**: This version of Calico does not support etcd v2.x; the upgrade steps will migrate
+  > your data from etcd v2 format to etcd v3 format.
+  {: .alert .alert-danger}
```

Comment:
I don't think it will be clear here whether we mean the etcd upgrade or the calico upgrade; it would be good to add a few more words to clarify that.

Perhaps: "This version of Calico requires etcd v3.x, because it has a more scalable DB access protocol.  Upgrading etcd will not change your existing Calico data, but will add that protocol capability.  The Calico upgrade steps below will then convert your existing data to take advantage of that capability."

Or is that too much?